src/sidebar/util/postmessage-json-rpc.js

```diff
@@ -33,7 +28,7 @@ export function call(
   params = [],
   timeout = 2000,
   window_ = window,
-  id = generateId()
+  id = generateHexString(10)
```

Comment:
I think we can just suppress the code coverage complaint here.

Reply:
I have removed the unnecessary test seam in a follow up commit.